crates/holochain_sqlite/src/db/kind.rs

```diff
@@ -20,12 +19,6 @@ pub enum DbKind {
     Conductor,
     /// Specifies the environment used to save wasm
     Wasm,
-    /// State of the p2p network (one per space).
-    #[display(fmt = "agent_store-{:?}", "_0")]
-    P2pAgentStore(Arc<KitsuneSpace>),
-    /// Metrics for peers on p2p network (one per space).
-    #[display(fmt = "metrics-{:?}", "_0")]
-    P2pMetrics(Arc<KitsuneSpace>),
```

Comment:
Removing the legacy agent store and p2p metrics databases now since they rely on types inside legacy kitsune.